deploy/azuredeploy.json

```diff
@@ -67,7 +67,8 @@
           },
           "properties": {
             "ResourceConnectionString": "[listkeys(variables('commsName'), '2020-08-20-preview' ).primaryConnectionString]",
```

Comment:
Is there a place to see the keys available in listKeys for CommunicationServices?
I could only find this which doesn't have much info https://docs.microsoft.com/en-us/azure/templates/microsoft.communication/2020-08-20/communicationservices?tabs=bicep

Reply:
Well more interested in understanding if there's more list* fns that could potentially be used to auto get the endpointUrl from the resource instead of parsing it out of the resource connection string
https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource#list

Reply:
The problem is that we are making this  resource as part of our steps. We can't know it ahead of time until its created. I think you can only create some of these variables at the very beginning.

Reply:
https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/variables
![image](https://user-images.githubusercontent.com/79329532/164500359-4f5b2d87-d530-4415-896e-cbedce9a8da8.png)